code/modules/organs/organ_internal.dm

```diff
@@ -19,7 +19,7 @@
 
 
 /datum/organ/internal/Destroy()
-	if(owner) //Just going to assume these lists are here because we have bigger problems than a runtime if they aren't
+	if(owner && owner.internal_organs.Find(src))
```

Comment:
```suggestion
	if(owner && (src in owner.internal_organs))
```
`in` is allegedly slightly faster, as `Find()` is a proc and also has more information to return